store/sqlstore/channel_store.go

```diff
@@ -1816,27 +1816,41 @@ func (s SqlChannelStore) GetMemberCountsByGroup(channelID string, includeTimezon
 	selectStr := "GroupMembers.GroupId, COUNT(ChannelMembers.UserId) AS ChannelMemberCount"
 
 	if includeTimezones {
-		distinctTimezones := `
-			DISTINCT(
-				CASE WHEN JSON_EXTRACT(Timezone, '$.useAutomaticTimezone') = 'true' AND LENGTH(Timezone) > 74
-				THEN JSON_EXTRACT(Timezone, '$.automaticTimezone')
-				WHEN LENGTH(Timezone) > 74
-				THEN JSON_EXTRACT(Timezone, '$.manualTimezone')
-				END
-			)
-		`
+		endOfFirstKey := `LOCATE(':', Users.Timezone) + 2`
+		lengthOfFirstValue := `LOCATE(',', Users.Timezone) - LOCATE(':', Users.Timezone) - 3`
+		endOfSecondKey := `LOCATE(',', Users.Timezone) + 19`
+		lengthOfSecondValue := `LOCATE('useAutomaticTimezone', Users.Timezone) - 22 - LOCATE(',', Users.Timezone)`
+
 		if s.DriverName() == model.DATABASE_DRIVER_POSTGRES {
-			distinctTimezones = `
-				DISTINCT(
-					CASE WHEN Timezone::json->>'useAutomaticTimezone' = 'true' AND LENGTH(Timezone) > 74
-					THEN Timezone::json->>'automaticTimezone'
-					WHEN LENGTH(Timezone) > 74
-					THEN Timezone::json->>'manualTimezone'
+			endOfFirstKey = `POSITION(':' IN Users.Timezone) + 2`
+			lengthOfFirstValue = `POSITION(',' IN Users.Timezone) - POSITION(':' IN Users.Timezone) - 3`
+			endOfSecondKey = `POSITION(',' IN Users.Timezone) + 19`
+			lengthOfSecondValue = `POSITION('useAutomaticTimezone' IN Users.Timezone) - 22 - POSITION(',' IN Users.Timezone)`
+		}
+
+		selectStr = `
+			GroupMembers.GroupId,
+			COUNT(ChannelMembers.UserId) AS ChannelMemberCount,
+			COUNT(DISTINCT
+				(
+					CASE WHEN Timezone like '%"useAutomaticTimezone":"true"}' AND LENGTH(Timezone) > 74
+					THEN
+					SUBSTRING(
+						Timezone
+						FROM ` + endOfFirstKey + `
+						FOR ` + lengthOfFirstValue + `
+					)
+					WHEN Timezone like '%"useAutomaticTimezone":"false"}' AND LENGTH(Timezone) > 74
+					THEN
+						SUBSTRING(
+						Timezone
+						FROM ` + endOfSecondKey + `
+						FOR ` + lengthOfSecondValue + `
+					)
```

Comment:
These numbers should be made into constants and possibly an explanation, so someone later could understand the magic numbers. 
```
const LENGTH_OF_DEFAULT_TIMEZONE = 74 
// len("{"automaticTimezone":"","manualTimezone":"","useAutomaticTimezone":"true"}")
```